src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/Source.cs

```diff
@@ -171,7 +173,8 @@ private static RoslynTrigger GetRoslynTrigger(AsyncCompletionData.CompletionTrig
                     return new RoslynTrigger(CompletionTriggerKind.Snippets);
             }
 
-            throw ExceptionUtilities.UnexpectedValue(trigger.Reason);
+            FatalError.ReportWithoutCrash(ExceptionUtilities.UnexpectedValue(trigger.Reason));
+            return RoslynTrigger.Invoke;
```

Comment:
i think it would be much better for us to just ignore anything we don't understand...